### DISCUSSION TITLE
Tldr no output.Help!

### DISCUSSION THREAD

**Discussion by 15222603763** [tldr-pages/tldr#13509] [Q&A]
My tldr doesn't work! No matter what tldr command I run, there is no any output, and the Error code is 0.I want to uninstall it. So could somebody teach me how to uninstall it? Or did someone meet similer question? Thanks :)

**Comment by kbdharun** [comment]
Hi, this seems to be an issue with the client you are using, can you provide me with a few details like the Operating system you are using, and how you installed tldr? (this will help me identify the client you are using so that I can help uninstall the client and suggest alternatives.)

**Reply by 15222603763** [reply]
Thank you for your help. I initially installed tldr on my Ubuntu system (Ubuntu 22.04.4 LTS on GNU/Linux 5.10.16.3-microsoft-standard-WSL2, x86_64) using apt, but it didn’t work. I then tried to upgrade it, but it still didn’t function properly, with the error message “no tldr entry for xxx.” After that, I installed it using npm, and it no longer displayed the “no entry for xxx” message. Instead, it produced no output at all, and the error code was 0, indicating that there were no errors. I was confused.

**Reply by kbdharun** [reply]
Using NPM in WSL is a bit problematic (if you have it installed in the host, that is given higher precedence). I would suggest installing our Python client using Snap for Ubuntu with the command `snap install tldr`. If the Snap command is not present, you can install it with `sudo apt install snapd`.

Alternatively, you can directly install it in Windows with the command `winget install tldr` (which installs our Rust client).
